docs/man_pages/lib-management/plugin-find.md

```diff
@@ -0,0 +1,26 @@
+plugin find
+==========
+
+Usage | Synopsis
+---|---
+Finds NativeScript plugins in npm | `$ tns plugin find [<PluginName>] [--all] [--count <Count>]`
```

Comment:
Finds NativeScript plugins in npm -> General

Unless there are multiple entries and combos of options which do different things, we usually go with General